src/components/Cards/Cards.js

```diff
@@ -1,28 +1,41 @@
 import Card from '../Card/Card';
 
-export default function Cards({exercises, filter}) {
+export default function Cards({onChangeBookmark, exercises, filter}) {
   const filteredExercises = exercises.filter(exercise => {
     if (filter === 'Alle') {
       return true;
     }
     if (filter === exercise.category) {
       return true;
+    }
+    if (filter === exercise.bookmarked) {
+      return true;
```

Comment:
This two if conditions could deliver some problems. If a exercise should filtered by category and it should be sorted out but is bookmarked it will be in the returned array.